stable/kubernetes-dashboard/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubernetes-dashboard
-version: 1.9.0
+version: 1.9.1
```

Comment:
1.10.1. Otherwise looks good.

Reply:
I updated the version number and pushed the change.